idaes/generic_models/properties/helmholtz/helmholtz.py

```diff
@@ -575,6 +640,20 @@ def build(self):
             doc="Smooth max parameter (pressure under)",
             units=pyunits.kPa)
 
+        # Unit conversion factors. Currently assume everything except external
+        # functions are on a molar basis with base SI units.  Most of these are
+        # used to convert quanties related to external functions.  Not using
+        # pyunits convert since these conversions are just prefixes.  Mass
+        # to moles is a simple conversions, since this is only one component
+        self.uc_J_per_mol_to_kJ_per_kg = pyunits.kJ/self.mw/1000/pyunits.J
+        self.uc_kJ_per_kg_to_J_per_mol = self.mw*pyunits.J*1000/pyunits.kJ
+        # Entropy type units are same as energy type units since K is left as is
+        self.uc_J_per_molK_to_kJ_per_kgK = self.uc_J_per_mol_to_kJ_per_kg
+        self.uc_kJ_per_kgK_to_J_per_molK = self.uc_kJ_per_kg_to_J_per_mol
+        # Pressure conversions
+        self.uc_kPa_to_Pa = 1000*pyunits.Pa/pyunits.kPa
+        self.uc_Pa_to_kPa = pyunits.kPa/pyunits.Pa/1000
+
```

Comment:
Yeah, that's right.  This is nice because it's fast and you can still check for consistency in the units.  The drawback is that you can't mix units like you could if we use convert.  Everything needs to be in the expected unts.